projects/valtimo/components/src/lib/components/page-title/page-title.component.scss

```diff
@@ -15,20 +15,53 @@
  */
 
 .page-title-container {
+  margin-bottom: 2rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+
   .page-title {
     font-weight: 300;
-    margin-bottom: 2rem;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
   }
 
-  .skeleton-container {
-    margin-bottom: 2rem;
+  .page-subtitle {
+    margin-left: 16px;
+    padding-bottom: 4px;
+    color: var(--cds-text-helper, #6f6f6f);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: 14.2px;
```

Comment:
is the .2 necessary? Same question for every decimal point in css. I doubt it makes that much of a difference and in the cases of heights, margins etc. I don't think the browser renders anything past the decimal point